README.md

```diff
@@ -38,39 +38,41 @@ IINA uses mpv for media playback. To build IINA, you can either fetch copies of
 ### Using the pre-compiled libraries
 
 1. Download pre-compiled libraries by running
+
 ```console
 ./other/download_libs.sh
 ```
+
   - Tips:
-    - Change URL in the shell script if you want to download arch-specific binaries. By default, it will download the universal ones. You can download other binaries from `https://iina.io/dylibs/${ARCH}/fileList.txt` where `ARCH` can be `universal`, `arm64` and `x86_64`.
-    - If you want to build an older IINA version, make sure to download the correponding dylibs. For example, `https://iina.io/dylibs/1.2.0/universal/fileList.txt`.
+    - Change the URL in the shell script if you want to download arch-specific binaries. By default, it will download the universal ones. You can download other binaries from `https://iina.io/dylibs/${ARCH}/fileList.txt` where `ARCH` can be `universal`, `arm64` and `x86_64`.
+    - If you want to build an older IINA version, make sure to download the corresponding dylibs. For example, `https://iina.io/dylibs/1.2.0/universal/fileList.txt`.
 
-2. Open iina.xcodeproj in the [latest public version of Xcode](https://itunes.apple.com/us/app/xcode/id497799835). *IINA may not build if you use any other version.*
+2. Open iina.xcodeproj in the [latest public version of Xcode](https://apps.apple.com/app/xcode/id497799835). *IINA may not build if you use any other version.*
 
 3. Build the project.
 
 ### Building mpv manually
 
-1. Build your own copy of mpv. If you're using a package manager to manage dependencies, the steps below outline the process.
+1. Build your copy of mpv. If you're using a package manager to manage dependencies, the steps below outline the process.
```

Comment:
Any reason for this change?

Reply:
Here's what Grammarly reports about it: "Own may be redundant. Redundant phrases such as final outcome, I personally, and short and concise express the same information twice. Avoiding this kind of redundancy will make your sentences sound tighter and more professional."

Reply:
I liked it as it was before.

Reply:
Reverted.